docs/tokens/nft-tokens.mdx

```diff
@@ -932,7 +932,9 @@ RegisterAndSetAllRolesDynamicTransaction {
     Data: "registerAndSetAllRolesDynamic" +
           "@" + <token name in hexadecimal encoding> +
           "@" + <token ticker in hexadecimal encoding> +
-          "@" + <token compressed type (which can be `NFT`, `SFT`, `META`, `FNG`) in hexadecimal encoding>
+          "@" + <token compressed type (which can be `NFT`, `SFT`, `META`) in hexadecimal encoding> +
+              # For META token type only (18 Decimals)
```

Comment:
I would remove `(18 Decimals)` from here because the denomination can be different than 18. I would rather add on L937, after the denominator something like `(number of decimals)`